pallets/marketplace/src/lib.rs

```diff
@@ -466,6 +563,14 @@ pub mod pallet {
                             JobStatus::Assigned,
                         );
                     }
+                    let total_jobs_assigned =
+                        <StoredTotalJobsAssigned<T>>::get(&reward_asset).unwrap_or_default();
+                    let new_total_jobs_assigned = total_jobs_assigned + T::AssetAmount::from(1);
```

Comment:
see above: this should be an unsigned integer not an AssetAmount

Reply:
agreed, fixed in https://github.com/Acurast/acurast-core/commit/3dfc18721e98aa5f02200efdefe7d42bbb67e5da